tests/php/ORM/SQLSelectTest.php

```diff
@@ -819,12 +842,12 @@ public function testBaseTableAliases()
         // In SS4 the "explicitAlias" would be ignored
         $query = SQLSelect::create('*', [
             'MyTableAlias' => '"MyTable"',
-            'explicitAlias' => ', (SELECT * FROM "MyTable" where "something" = "whatever") as "CrossJoin"'
+            'explicitAlias' => '(SELECT * FROM "MyTable" where "something" = "whatever") as "CrossJoin"'
         ]);
         $sql = $query->sql();
 
         $this->assertSQLEquals(
-            'SELECT * FROM "MyTable" AS "MyTableAlias" , ' .
+            'SELECT * FROM "MyTable" AS "MyTableAlias", ' .
```

Comment:
This test was only here to test whether _aliases_ were working the way they were expected to. The added comma was just to work around this arbitrary limitation to get the test to work, rather than explicitly testing that you had to add a comma for subsequent tables.